src/fhirtypes/ElementDefinition.ts

```diff
@@ -324,10 +324,10 @@ export class ElementDefinition {
         // @ts-ignore
         return prop && !isEqual(this[prop], original[prop]);
       }) ||
-      // When a slice has children that changed, we must treat the slice as if it
+      // When a slice or a sliced element has children that changed, we must treat the slice as if it
```

Comment:
As a minor clarification: `...we must treat the slice or sliced element as if it...`